### PR TITLE
Update the readme roadmap link to use 2022 one

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:

**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- updates the readme roadmap link to use 2022 one

### Affected modules and functionalities:
- README

### Key points relevant for the review:
- NA

### Tests
- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [x] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [ ] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: README.rst [hunk 1]
```diff
@@ -102,7 +102,7 @@ We strongly encourage you to comment on our roadmap and provide us feedback on t
 GitHub issue.
 
 .. |dali-roadmap-link| replace:: The following issue represents
```
Comment (carmocca) [inline README.rst:104]: Should update the year in L98 too

**Review by carmocca — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

File: README.rst (hunk 1 above)
Comment (JanuszL) [reply README.rst:104]: Fixed

**Comment by JanuszL** [comment]
!build

**Review by szalpal — APPROVED** [review]

**Pull request merged**